### PR TITLE
Admin program index view adjustments for smaller screen sizes in new UI

### PR DESCRIPTION
**Pull request by clouser2**
### Description
Adding some media breakpoint adjustments for smaller screensizes, inlcluding:
* Rendering a shorter date form (e.g. "Published on 2022/11/10" instead of "Published on 2022/11/10 at 09:00am PDT")
* Smaller text size
* Adjusting item padding

Also moved time formatting code from `BaseHtmlView` to `DateConverter`.

Before:
Screen Shot 2022-06-13 at 3 42 03 PM

After:
Screen Shot 2022-06-13 at 3 40 04 PM

### Issue(s)
#1238

### CODE REVIEW

File: server/app/views/BaseHtmlView.java [hunk 1]
```diff
@@ -99,6 +99,11 @@ protected String renderDateTime(Instant time, ZoneId zoneId) {
     return dateTime.format(DateTimeFormatter.ofPattern("yyyy/MM/dd 'at' h:mm a z"));
   }
 
+  protected String renderDate(Instant time, ZoneId zoneId) {
+    ZonedDateTime dateTime = time.atZone(zoneId);
+    return dateTime.format(DateTimeFormatter.ofPattern("yyyy/MM/dd"));
+  }
+
```
Comment (bion) [inline server/app/views/BaseHtmlView.java:106]: suggestion (blocking): what do you think of moving this into `services.DateConverter`?

**Review by bion — CHANGES_REQUESTED** [review]

File: server/app/views/BaseHtmlView.java (hunk 1 above)
Comment (clouser2) [reply server/app/views/BaseHtmlView.java]: Done.

Comment (bion) [reply server/app/views/BaseHtmlView.java]: nice, thanks!

File: server/app/views/admin/programs/ProgramAdministratorProgramListView.java [hunk 2]
```diff
@@ -12,11 +12,11 @@
 import com.typesafe.config.Config;
 import controllers.admin.routes;
 import j2html.tags.Tag;
-import java.time.ZoneId;
 import java.util.List;
 import java.util.Optional;
 import javax.inject.Inject;
 import play.twirl.api.Content;
+import services.DateConverter;;
```
Comment (bion) [inline server/app/views/admin/programs/ProgramAdministratorProgramListView.java:19]: double semi-colons here

**Review by bion — APPROVED** [review]

File: server/app/views/admin/programs/ProgramAdministratorProgramListView.java (hunk 2 above)
Comment (clouser2) [reply server/app/views/admin/programs/ProgramAdministratorProgramListView.java:19]: Fixed.

**Pull request merged**